python/src/LsqSolver_doc.i.in

```diff
@@ -0,0 +1,55 @@
+%feature("docstring") OT::LsqSolver
+"Least Square Solver (LsqSolver) for a system of non-linear equations.
+
+Available constructor:
+    LsqSolver()
```

Comment:
no need to document the default ctor, so you can drop this "Available constructor"